tests/utils/test_ensemble.py

```diff
@@ -261,6 +284,7 @@ def test_compile():
     # base_model, keras_model, generator, train_gen
     gnn_models = [
         create_graphSAGE_model(graph),
```

Comment:
For consistency, might just rename `create_graphSAGE_model` everywhere as `create_GraphSAGE_model`. Again, minor point.

Reply:
Done.